example_dags/example_append_and_replace_strategies.py

```diff
@@ -0,0 +1,69 @@
+import os
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.snowflake.operators.snowflake import SnowflakeOperator
+
+from universal_transfer_operator.constants import FileType
+from universal_transfer_operator.datasets.file.base import File
+from universal_transfer_operator.datasets.table import Table
+from universal_transfer_operator.universal_transfer_operator import (
+    TransferIntegrationOptions,
+    UniversalTransferOperator,
+)
+
+s3_bucket = os.getenv("S3_BUCKET", "s3://astro-sdk-test")
+gcs_bucket = os.getenv("GCS_BUCKET", "gs://uto-test")
+transform_table = """
+    SELECT
+       *
+    FROM {{ params.table_name }}
+"""
+
+with DAG(
```

Comment:
Would it make sense to have an example for the Local filesystem/SFTP and SQLite? mainly because they will be quick to execute.

Reply:
[Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/45/commits/cc34d5caaf1884552e94c9a513f1f8410ba8c68c)